itou/utils/validators.py

```diff
@@ -30,3 +31,8 @@ def validate_pole_emploi_id(pole_emploi_id):
                 "7 chiffres suivis d'une 1 lettre ou d'un chiffre."
             )
         )
+
+
+def validate_birthdate(birthdate):
+    if birthdate >= timezone.now().date():
+        raise ValidationError(_("Cette date de naissance n'est pas valide"))
```

Comment:
Thank you :) Peut-être expliquer à l'utilisateur pourquoi elle n'est pas valide ? => elle est dans le futur.

Reply:
Oui un peu plus explicite serait bien, et voir si on pourrait pas définir un age minimal pour un candidat